source/simulator/simulator_access.cc

```diff
@@ -211,6 +211,13 @@ namespace aspect
   }
 
   template <int dim>
+  bool
+  SimulatorAccess<dim>::include_melt_transport () const
```

Comment:
I am a bit hesitant to add a lot of these functions to `SimulatorAccess`, because they all could be replaced by calls to `this->get_parameters().include_melt_transport`. I see that this function is called often enough to justify the tiny simplification, but for many other cases I am not a big fan.